lib/src/knobs/knobs.dart

```diff
@@ -33,6 +34,9 @@ abstract class KnobsBuilder {
   /// Creates text input field with [label] and [initial] value.
   String text(String label, {String initial = ''});
 
+  double slider(String label,
+      {double initial = 0, double max = 100, double min = 0});
```

Comment:
I'd say it's better to use `1` as a default `max` value, as it's compliant with default values for a slider.